apps/01PigLatin.js

```diff
@@ -6,11 +6,82 @@ prompt.start();
 
 
 function pigLatin(word) {
+  // Concept Check
+  //
+  // Please explain why we are using String.prototype.toLowerCase() here.
+  // For more info:
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase
+  //
+  // Your Answer: Since the .indexOf() method is case sensitive, we are using String.prototype.toLowerCase() here to account for string inputs with problematic Capitalization / Lowercase schemes. This way the function will not be disrupted by string inputs that contain some letters Capitalized, some letters Lowercase, as it will simply use 'word = word.toLowerCase();' to convert the string to all Lowercase before continuing to pass the program.
+  //
```

Comment:
.indexOf() is not case sensitive. Strings, are by their very nature case sensitive.

Close tho.